### PR TITLE
[MIG] purchase_operating_unit: Migration to 17.0

### PR DESCRIPTION
**Pull request by ghost**
Depends on:

- [x] https://github.com/OCA/operating-unit/pull/667
- [x] https://github.com/OCA/operating-unit/pull/669
- [x] https://github.com/OCA/operating-unit/pull/670

### CODE REVIEW

File: purchase_operating_unit/security/purchase_security.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+<!--Copyright 2015-17 ForgeFlow S.L.-->
+<!-- - Jordi Ballester Alomar-->
+<!--Copyright 2015-17 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya-->
+<!--License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo noupdate="1">
+    <record id="ir_rule_purchase_order_allowed_operating_units" model="ir.rule">
+        <field name="model_id" ref="purchase.model_purchase_order" />
+        <field
+            name="domain_force"
+        >['|',('operating_unit_id','=',False),('operating_unit_id','in', user.operating_unit_ids.ids)]</field>
```
Comment (traitrantrobz) [inline purchase_operating_unit/security/purchase_security.xml]: @jdidderen-nsi I think you should double-check this place using this link https://github.com/OCA/operating-unit/pull/667#discussion_r1629838296

**Comment by traitrantrobz** [comment]
Except `purchase_security.xml`, everything else is look good.

File: purchase_operating_unit/security/purchase_security.xml (hunk 1 above)
Comment (ghost) [reply purchase_operating_unit/security/purchase_security.xml]: Fixed 👍

File: test-requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+odoo-addon-analytic_operating_unit @ git+https://github.com/OCA/operating-unit.git@refs/pull/669/head#subdirectory=analytic_operating_unit
```
Comment (traitrantrobz) [inline test-requirements.txt]: The module `analytic_operating_unit`has been merged. Please update the test-requirements.txt file.

**Review by traitrantrobz — CHANGES_REQUESTED** [review]

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**